packages/origin-ui-core/src/components/Organization/OrganizationInvite.tsx

```diff
@@ -72,6 +74,13 @@ export function OrganizationInvite() {
 
             if (error?.response?.status === 401) {
                 showNotification('Unauthorized.', NotificationType.Error);
+            } else if (error.toJSON().message === 'Request failed with status code 412') {
```

Comment:
Can't we use `error.response.status === 412` like above?
Comparing strings in errors seems hacky.